goss-testing/scripts/check_bios_firmware_versions.sh

```diff
@@ -57,8 +57,38 @@ function set_vars() {
   fi
 
   # Allow for output of documentation when needed
+  rc=0
+
+  DL325_FW="2.46"
+  DL385_FW="2.46"
+
+  DL325_BIOS="v2.40"
+  DL385_BIOS="v2.40"
+
+  # Taken from HFP-firmware-22.05 but leaving here for reference in this code
+  # Appropriate versions to use for Shasta 1.5.0 or HPCM as of 2/2/2022 unless otherwise indicated by your
```

Comment:
Do we want to use Shasta 1.5 here, since this is the csm-1.2 branch?

Reply:
That comment was direct from the HFP notes.  I changed it to csm 1.2